ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1077,3 +1076,63 @@
                     default-excludes: true
                     fingerprint: false
                     only-if-success: false
+      - '{name}-{test_name}-for-period':
+          test_name: daily-capa-aws-clean-up
+          node: 'capa-test'
+          description: 'This is for cleaning up old AWS capa clusters daily.'
+          builders:
+            - builder-capa-aws-clean-up
+          branches:
+          - '*/main'
+          included_regions: []
+          cron: '' # 'H H * * *'
```

Comment:
Can you explain why it's `'' # 'H H * * *'` ? I thought it should be `'H H * * *'`. Same question for `cron` on line 1115.